test/test_playbooks/tasks/activation_key.yml

```diff
@@ -1,4 +1,5 @@
 ---
+- debug: var=activation_key_host_collections
```

Comment:
Please remove this debug statement again.

Reply:
Oops. 🤦‍♂